bazel/grpc_build_system.bzl

```diff
@@ -240,6 +254,42 @@ def grpc_package(name, visibility = "private", features = []):
             features = features,
         )
 
+def grpc_objc_library(
```

Comment:
Did you preemptively add this function in our internal version so you won't break the next import after merging this?

Reply:
Yeah. Should've to let you know beforehand: https://critique.corp.google.com/#review/258434245&tab=a

Reply:
`grpc_objc_library` has been merged in Google3 as well by CL 258434245